### ISSUE TITLE
[Task] Support pre-trained vector embeddings as input features 

### ISSUE THREAD

**Issue by sararb** [NVIDIA-Merlin/Transformers4Rec#485]
### Description
Provide support for pretrained embeddings passed from the data loader

### Goals
- [ ] Update the  [input block](https://github.com/NVIDIA-Merlin/Transformers4Rec/blob/21a2a836a30610746bc8788ff926bad74fd00289/transformers4rec/torch/features/sequence.py#L97) to support the pre-trained embeddings:
      - Add a module [to_merge](https://github.com/NVIDIA-Merlin/Transformers4Rec/blob/21a2a836a30610746bc8788ff926bad74fd00289/transformers4rec/torch/features/tabular.py#L75%23L81) to extract pre-trained embeddings

- [ ] Show an example of how to transform pre-trained embeddings before aggregating them with the other trainable embeddings in the model. Such as `Layer Normalization` or `MLP layer`. 

- [ ] Implement and evaluate different aggregation methods to combine pre-trained embeddings with the other trainable embeddings.

### Constraints
- Dataloader should support fixed-size 3D tensors. 
- Dataloader needs to add a tag to the pre-trained embeddings so that the T4R input module can differentiate between trainable features and pre-trained embeddings.

**Comment by gabrielspmoreira** [comment]
When the embedding table are not huge and fit GPU memory, the new `PretrainedEmbeddingsInitializer` ( https://github.com/NVIDIA-Merlin/Transformers4Rec/pull/572 ) can be used to initialize the embedding matrix with pre-trained embeddings and set them to trainable or not.

**Comment by sararb** [comment]
closed by #690
